src/js/content_scripts/input-mapper.js

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+*/
+
+/* eslint-env browser */
```

Comment:
I would suggest setting this in your `.eslintrc.json` so that you only need to specify the environment for things like the `Gruntfile.js` that run in Node.